BO4E/BO/Zeitfenster.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Text.Json.Serialization;
+
+using Newtonsoft.Json;
+
+using ProtoBuf;
+
+namespace BO4E.COM
+{
+    /// <summary>
+    /// Ein Zeitfenster mit Start- und Endzeiten mit <see cref="TimeOnly"/> als Typ.
+    /// </summary>
+    [ProtoContract]
+    public class Zeitfenster : COM
+    {
+        /// <summary>Startzeit</summary>
+        [JsonProperty(Required = Required.Default, Order = 3, PropertyName = "startzeit")]
+        [JsonPropertyName("startzeit")]
+        [JsonPropertyOrder(3)]
+        [ProtoMember(3)]
+        public TimeOnly? Startzeit { get; set; }
+
+        /// <summary>Endzeit</summary>
+        [JsonProperty(Required = Required.Default, Order = 4, PropertyName = "endzeit")]
+        [JsonPropertyName("endzeit")]
+        [JsonPropertyOrder(4)]
+        [ProtoMember(4)]
+        public TimeOnly? Endzeit { get; set; }
+
+        /// <summary>
+        /// Erstellt eine neue Instanz von Zeitfenster mit spezifischen Start- und Endzeiten.
+        /// </summary>
+        /// <param name="startzeit">Startzeit (Format: HHMM)</param>
+        /// <param name="endzeit">Endzeit (Format: HHMM)</param>
```

Comment:
```suggestion
        /// <param name="startzeit">inklusive Startzeit</param>
        /// <param name="endzeit">exklusive Endzeit</param>
```
an der stelle ist das format ja egal, weil es kein string ist.